src/lib.zig

```diff
@@ -1190,7 +1190,7 @@ pub fn Server(comptime Context: type) type {
 
                 const entry = try self.registered_files.getOrPut(client.response_state.file.path);
                 if (!entry.found_existing) {
-                    entry.key_ptr.* = try self.root_allocator.dupeZ(u8, client.response_state.file.path);
+                    entry.key_ptr.* = try self.root_allocator.dupe(u8, client.response_state.file.path);
```

Comment:
I'm not entirely sure why this used `dupeZ` since I don't think it's being passed to any `C` API that expects null terminated strings. I changed it to use `dupe` in order to avoid tests failing on account of https://github.com/ziglang/zig/issues/17668. Let me know if I missed something.

Reply:
I have no idea to be honest, I wrote that a long time ago :) It does seem like there's no reason for this code to use `dupeZ`